book/src/volume-cloning.md

```diff
@@ -0,0 +1,45 @@
+# Volume Cloning
+
+## Status
+
+* Kubernetes 1.15: Alpha
+
+## Overview
+
+A storage provider that allows volume cloning as a create feature, may choose to implement volume cloning via a
+control-plane CSI RPC call.
+
+## Implementing Volume cloning functionality
+
+To implement volume cloning the CSI driver MUST:
+
+1. Implement checks in `CreateVolume` function implementation.
```

Comment:
What parameters are set in CreateVolume that indicate this should be a clone?  This should be detailed in this section

Reply:
Line #21 doesn't cut it?  I'll rework it.